.github/workflows/check_label.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Release Notes label
         run: |
           if [[ "${{contains( github.event.pull_request.labels.*.name, 'release notes (needs details)')}}" == "true" ]]; then
-            echo The "release notes (needs details)" label is set. The changes made in this Pull Request need to be documented in the release notes summary "('./doc/releasenotes/16_0_0_summary.md')". Once documented, the "release notes (needs details)" label can be removed.
+            echo The "release notes (needs details)" label is set. The changes made in this Pull Request need to be documented in the release notes summary "('./changelog/17.0/17.0.0/17_0_0_summary.md')". Once documented, the "release notes (needs details)" label can be removed.
```

Comment:
This feels like a lot of redundant info/context for the file: 17.0 in 3 places, 17.0.0 in 2 places. Would this e.g. not be good for any reasons? `./changelog/17.0.0/summary.md`?

Reply:
Sounds good, fixed via 88419a3411e2cd45f924aaa15e196995b74085cd. I kept `17.0/17.0.0` but removed the prefix of the different READMEs.